cli/azd/pkg/project/framework_service_docker.go

```diff
@@ -200,6 +201,18 @@ func (p *dockerProject) Build(
 		return &ServiceBuildResult{Restore: restoreOutput}, nil
 	}
 
+	// if java project no Dockerfile, add a default one for Docker build
```

Comment:
nit: `if java project no Dockerfile` -> `if it's a java project without Dockerfile`